test/jdk/java/foreign/callarranger/TestLayoutEquality.java

```diff
@@ -26,7 +26,7 @@
  * @test
  * @enablePreview
  * @compile platform/PlatformLayouts.java
- * @modules java.base/jdk.internal.foreign
+ * @modules java.base/jdk.internal.foreign java.base/jdk.internal.foreign.abi
```

Comment:
Is the first module export still required? Seems to me that only `foreign.abi` is needed (looking at the imports)

Reply:
Also, please check the import - I see this in the file:

```
import platform.PlatformLayouts;
```

This doesn't see to make sense?

Reply:
Ah - nvm, I missed that file has been moved to a test-private package.